### PR TITLE
[infra] Added multi-agent type support (in theory)

### PR DESCRIPTION
**Pull request by Razvan00Rusu**

### CODE REVIEW

**Review by neeldug — APPROVED** [review]
LGTM

File: pkg/infra/main.go [hunk 1]
```diff
@@ -15,9 +15,14 @@ import (
 	"math"
 	"math/rand"
 
+	"github.com/google/uuid"
 	"github.com/joho/godotenv"
 )
 
+var InitAgentMap = map[string]agent.Strategy{
```
[the remark this answers is not in the document]
Comment (Razvan00Rusu) [reply pkg/infra/main.go:22]: Agents wouldn't have access to this so maybe not necessary?

[the remark this answers is not in the document]
Comment (neeldug) [reply pkg/infra/main.go:22]: Though I suppose that makes it way chunkier... Could just leave a comment to ensure no-one fiddles with it later

**Review by 0x6770 — APPROVED** [review]
LGTM

**Pull request merged**